docker-compose.dev.yml

```diff
@@ -39,6 +39,7 @@ services:
     tty: true        # docker run -t
 
   localstack:
+    image: civiform-localstack
```

Comment:
This should be done in the main docker-compose.yml, unless you are trying to override it.

Reply:
Yah, the choice to override was explicit. There's no need to install this hook for other non-local workstation runs (e.g. CI). Added a comment.